README.md

```diff
@@ -586,6 +586,77 @@ crud.len('customers')
 ...
 ```
 
+### Statistics
+
+`crud` routers can provide statistics on called operations.
+```lua
+-- Enable statistics collect.
+crud.enable_stats()
+
+-- Returns table with statistics information.
+crud.stats()
+
+-- Returns table with statistics information for specific space.
+crud.stats('my_space')
+
+-- Disables statistics collect and destroys all collectors.
+crud.disable_stats()
+
+-- Destroys all statistics collectors and creates them again.
+crud.reset_stats()
+```
+
+Format is as follows.
+```lua
+crud.stats()
+---
+- spaces:
+    my_space:
+      insert:
+        ok:
+          latency: 0.002
+          count: 19800
+          time: 39.6
+        error:
+          latency: 0.000001
+          count: 4
+          time: 0.000004
+  space_not_found: 0
```

Comment:
There are automatic schema reload in crud. Probably we could add stats for it as well https://github.com/tarantool/crud/commit/5fa62ae3b621be853ad3b9c54aa6a04c95008c47

Reply:
Added in new commit